File: Fundamentos-Linguagens-UFC/06-subprogramas/subprogramas.c
#include <stdio.h>

void dobrar_valor(int n) {
    n = n * 2;
    printf("Dentro da função: %d\n", n);
}

int main() {
    int x = 5;
    printf("Antes da função: %d\n", x);
    dobrar_valor(x);
    printf("Depois da função: %d\n", x);
    return 0;
}

File: Fundamentos-Linguagens-UFC/06-subprogramas/subprogramas.py
def dobrar_elementos(lista):
    for i in range(len(lista)):
        lista[i] *= 2
    print("Dentro da função:", lista)

valores = [1, 2, 3]
print("Antes da função:", valores)
dobrar_elementos(valores)
print("Depois da função:", valores)
